scripts/exgdas_enkf_earc_vrfy.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python3
+
+import os
+
+from pygfs.task.archive import Archive
+from wxflow import AttrDict, Logger, cast_strdict_as_dtypedict, chdir, logit
+
+# initialize root logger
+logger = Logger(level=os.environ.get("LOGGING_LEVEL", "DEBUG"), colored_log=True)
+
+
+@logit(logger)
+def main():
+
+    config = cast_strdict_as_dtypedict(os.environ)
+
+    # Instantiate the Archive object
+    archive = Archive(config)
+
+    # Pull out all the configuration keys needed to run the rest of archive steps
+    keys = ['ATARDIR', 'current_cycle', 'IAUFHRS', 'RUN', 'PDY',
+            'PSLOT', 'HPSSARCH', 'LOCALARCH', 'ROTDIR', 'PARMgfs',
+            'ARCDIR', 'SDATE', 'MODE', 'ENSGRP', 'NMEM_EARCGRP',
+            'NMEM_ENS', 'DO_CALC_INCREMENT_ENKF_GFS', 'DO_JEDIATMENS',
+            'lobsdiag_forenkf', 'FHMIN_ENKF', 'FHMAX_ENKF_GFS',
+            'FHOUT_ENKF_GFS', 'FHMAX_ENKF', 'FHOUT_ENKF', 'ENKF_SPREAD',
+            'restart_interval_enkfgdas', 'restart_interval_enkfgfs',
+            'DOHYBVAR', 'DOIAU_ENKF', 'IAU_OFFSET', 'DOIAU', 'DO_CA',
+            'DO_CALC_INCREMENT', 'assim_freq', 'ARCH_CYC', 'DO_JEDISNOWDA',
+            'ARCH_WARMICFREQ', 'ARCH_FCSTICFREQ',
+            'IAUFHRS_ENKF', 'NET', 'NMEM_ENS_GFS']
```

Comment:
I think this list can be pared down significantly.  Just a guess:

```suggestion
    keys = ['current_cycle', 'RUN', 'PSLOT', 'ROTDIR', 'PARMgfs',
            'ARCDIR', 'MODE', 'DO_JEDIATMENS', 'DO_JEDISNOWDA']